src/plantify_apps/src/components/startup/steps/BasicInformationStep.tsx

```diff
@@ -0,0 +1,284 @@
+          </label>
+          <input
+            type='text'
+            name='startupName'
+            value={formData.startupName || ''}
+            onChange={handleChange}
+            placeholder='Enter your startup name here'
+            className={`${inputStyle} ${errors.startupName ? 'border-red-500' : ''}`}
+            required
+          />
+          {errors.startupName && (
+            <p className='mt-1 text-sm text-red-600'>{errors.startupName}</p>
+          )}
+        </div>
+
+        {/* Startup Logo */}
+        <div className='space-y-2'>
+          <FileUpload
+            label='Startup Logo'
+            accept='.jpg,.png,.pdf'
+            maxSize='2MB'
+            fileTypes='jpg, png, or pdf'
+            onFileSelect={handleLogoUpload}
+            disabled={isUploading}
+          />
+
+          {isUploading && (
+            <div className='flex items-center space-x-2 text-blue-600 mt-2'>
+              <Loader2 size={16} className='animate-spin' />
+              <span className='text-sm'>Uploading logo...</span>
+            </div>
+          )}
+
+          {logoPreviewUrl && !isUploading && (
+            <div className='mt-2'>
+              <p className='text-sm font-medium text-gray-700 mb-1'>Preview:</p>
+              <div className='flex items-center space-x-2'>
+                <div className='h-16 w-16 rounded-md overflow-hidden border border-gray-200'>
+                  <img
+                    src={logoPreviewUrl}
+                    alt='Logo Preview'
+                    className='h-full w-full object-cover'
+                  />
+                </div>
+                <div className='text-sm text-green-600 flex items-center'>
+                  <svg
+                    className='w-4 h-4 mr-1'
+                    fill='none'
+                    stroke='currentColor'
+                    viewBox='0 0 24 24'
+                    xmlns='http://www.w3.org/2000/svg'
+                  >
+                    <path
+                      strokeLinecap='round'
+                      strokeLinejoin='round'
+                      strokeWidth={2}
+                      d='M5 13l4 4L19 7'
+                    />
+                  </svg>
+                  Uploaded successfully
+                </div>
+              </div>
+            </div>
+          )}
+        </div>
+
+        {/* Business Sector and Founded Year */}
+        <div className='grid grid-cols-1 md:grid-cols-2 gap-6'>
+          <div>
+            <label className='block text-sm font-medium text-gray-700 mb-2'>
+              Business sector <span className='text-red-500'>*</span>
+            </label>
+            <select
+              name='sector'
+              value={formData.sector || ''}
+              onChange={handleChange}
+              className={`${inputStyle} ${errors.sector ? 'border-red-500' : ''}`}
+              required
+            >
+              <option value=''>Select sector</option>
```

Comment:
Please use constants for sector, so the options only map the arrays, for example

`const SECTORS = [
{
value: fintech,
label: FINTECH
}
]`